projects/ontimize-web-ngx/src/lib/components/table/o-table.component.ts

```diff
@@ -487,7 +487,7 @@ export class OTableComponent extends OServiceComponent implements OnInit, OnDest
   protected clickTimer;
   protected clickDelay = 200;
   protected clickPrevent = false;
-  protected editingCell: any;
+  public editingCell: any;
   protected editingRow: any;
```

Comment:
Why this attributes are not boolean?

Reply:
editingCell and editingRow does not contain a boolean value, those variables contains information of the cell/row being edited. Maybe a refactor is needed